ui/src/containers/UserProfile/index.tsx

```diff
@@ -15,16 +15,48 @@ import './UserProfile.css';
 const UserProfile: React.FC = () => {
   const { user } = useMst();
 
+  const [refreshId, setRefreshId] = useState<number>();
```

Comment:
Here what will the initial value of `refreshId`

Reply:
undefined

Reply:
Then i think it is not right to have `undefined` values as by default

Reply:
done !